ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot

```diff
@@ -27,26 +26,21 @@ Run Data Science Pipelines Operator Integration Tests
     ...     ODS-2632    AutomationBug
     ${openshift_api}    Get Openshift Server
     Log    ${openshift_api}
-    ${return_code}    ${output}    Run And Return Rc And Output    cd ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR} && make integrationtest K8SAPISERVERHOST=${openshift_api} DSPANAMESPACE=${DSPANAMESPACE} KUBECONFIGPATH=${KUBECONFIGPATH}
+    ${return_code}    ${output}    Run And Return Rc And Output    cd ${DSPO_SDK_DIR} && GIT_WORKSPACE=${DSPO_SDK_DIR} sh .github/scripts/tests/tests.sh --rhoai --k8s-api-server-host ${openshift_api} --kube-config ${KUBECONFIGPATH} --dspa-path ${DSPO_SDK_DIR}/tests/resources/dspa.yaml --external-dspa-path ${DSPO_SDK_DIR}/tests/resources/dspa-external.yaml --clean-infra --endpoint-type route
     Log    ${output}
     Should Be Equal As Integers	   ${return_code}	 0  msg= Run Data Science Pipelines Operator Integration Tests failed
 
 #robocop: disable: line-too-long
 *** Keywords ***
 Prepare Data Science Pipelines Operator Integration Tests Suite
     [Documentation]    Prepare Data Science Pipelines Operator Integration Tests Suite
-    ${return_code}    ${output}     Run And Return Rc And Output    rm -fR ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR}
+    ${return_code}    ${output}     Run And Return Rc And Output    rm -fR ${DSPO_SDK_DIR}
     Log    ${output}
-    ${return_code}    ${output}     Run And Return Rc And Output    git clone ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_URL} ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR}
+    ${return_code}    ${output}     Run And Return Rc And Output    git clone ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_URL} ${DSPO_SDK_DIR}
     Log    ${output}
-    Should Be Equal As Integers	   ${return_code}	 0  msg=Unable to clone data-science-pipelines-operator repo ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_URL}:${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_BRANCH}:${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR}
-    ${return_code}    ${output}     Run And Return Rc And Output    cd ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR} && git checkout -b it_test origin/${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_BRANCH}
+    Should Be Equal As Integers	   ${return_code}	 0  msg=Unable to clone data-science-pipelines-operator repo ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_URL}:${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_BRANCH}:${DSPO_SDK_DIR}
+    ${return_code}    ${output}     Run And Return Rc And Output    cd ${DSPO_SDK_DIR} && git checkout -b it_test origin/${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_BRANCH}
     Should Be Equal As Integers	   ${return_code}	 0  msg=Unable to checkout data-science-pipelines-operator
     RHOSi Setup
-    ${rc}    ${out}=    Run And Return Rc And Output    oc new-project ${DSPANAMESPACE}
-    Should Be Equal As Integers	   ${rc}	 0  msg=Cannot create a new project ${DSPANAMESPACE}
-
-Teardown Data Science Pipelines Operator Integration Tests Suite
```

Comment:
is the teardown implemented in the https://github.com/red-hat-data-services/ods-ci/pull/1833/files#diff-0639f9a87a08627ad0f0742ca7d12ea4ac9ad2a21072e8d216a04d60f43994b9R29 itself?

Reply:
Yes. We delegate to the DSPO test suite.